frontend/components/development/style-page.tsx

```diff
@@ -0,0 +1,145 @@
+import React, { Fragment } from 'react'
```

Comment:
will this page be enabled in production?

Reply:
Nope. Its a bit buried, but if you have a look at [gatsby-node.js](https://github.com/badges/shields/pull/4756/files#diff-fda05457e393bada716f508859bfc604) its conditional on the `INCLUDE_DEV_PAGES` env var which is off in production. Same with `frontend/components/development/logo-page.tsx`.